docs/evaluation/examples.rst

```diff
@@ -261,6 +264,13 @@ show the "raw" data (you can see that each group has at least
 
 .. image:: ../_static/evaluation/pha_data_manual.png
 
+.. note::
+   The :py:meth:`~sherpa.astro.data.DataPHA.apply_filter` and
+   :py:meth:`~sherpa.astro.data.DataPHA.apply_grouping` and
+   methods have changed in the 4.14 release: the grouping
```

Comment:
Is 4.14 release number consistent with our planned release versions?

Reply:
Technically it's 4.14.0 but I think it's okay for these documents to just say 4.14. We are not getting a 4.13.2 release.